tests/compile-fail/stacked_borrows/illegal_write4.rs

```diff
@@ -0,0 +1,31 @@
+// The compiler inserts some reborrows, enable optimizations to
+// get rid of them.
+// compile-flags: -Zmir-opt-level=1
+
+use std::mem;
+
+// This is an example of a piece of code that intuitively seems like we might
+// want to reject it, but that doesn't turn out to be possible.
+
+fn main() {
+    let target = 42;
+    // Make sure a cannot use a raw-tagged `&mut` pointing to a frozen location, not
+    // even to create a raw.
+    let r#ref = &target; // freeze
```

Comment:
to the best of my knowledge, the point of raw identifiers was explicitly not allowing the use of existing keywords as identifiers, but to enable forward compatibility even in the presence of newly introduced keywords.

Reply:
But these are references, why wouldn't I call them `ref`? :P

I'll change them back. I still totally would do this again in my own project though :D

Reply:
I totally love them. I just... don't know if we should be using them